src/main/java/seedu/address/logic/commands/DeleteLessonCommand.java

```diff
@@ -93,7 +94,7 @@ private static ModuleClass createModifiedModuleClass(ModuleClass targetModuleCla
         Set<UUID> studentsIds = new HashSet<>(targetModuleClass.getStudentUuids());
         List<Lesson> listOfLessons = targetModuleClass.getLessons();
         List<Lesson> editedListOfLessons = new ArrayList<>();
-        for (Lesson lesson:listOfLessons) {
+        for (Lesson lesson: listOfLessons) {
```

Comment:
Maybe keep the original?

Reply:
I think I will stick with the space, because it follows the comma then space convention. 
For example: `new A(a, b, c)`. So it feels more natural to have the space after colon.

Reply:
Then we might have to do a full refactor for all occurrences and stick with it? Because from what i remember, many of the shorthand for loops do not have spaces

Reply:
I think typically there are spaces both before and after the `:` in enhanced for loops?

Reply:
Changed to spaces before and after colons